ui/js/models/resultset.js

```diff
@@ -267,7 +267,12 @@ treeherder.factory('ThResultSetModel', ['$http', '$location',
                     const graph = resp.data;
 
                     // Build a mapping of buildbot buildername to taskcluster tasklabel for bbb tasks
-                    const builderToTask = _.omit(_.invert(_.mapValues(graph, 'task.payload.buildername')), [undefined]);
+                    const builderToTask = {};
+                    Object.entries(graph).forEach(([key, value]) => {
```

Comment:
This would be better done with a ``reduce`` rather than a ``forEach``.  Would you please use that function instead?  Thanks!!  :)